constants.py

```diff
@@ -98,6 +98,13 @@ def error_string(error, logname, with_hd):
 logsfs_label_prefix = 'logs-'
 logsfs_label = logsfs_label_prefix + '%s'
 
+rootpart_label = "root"
+backuppart_label = "backup"
+lvmpart_label = "Local SR"
```

Comment:
This label might be used on the kernel command-line (and in `/dev/disk/by-partlabel`) so using spaces can make it a bit awkward. I'd suggest `localsr`.